src/frontend/src/flows/register/index.ts

```diff
@@ -62,6 +70,10 @@ export const registerFlow = async <T>({
       const result = await register({
         identity,
         alias,
+        keyType: authenticatorAttachmentToKeyType(
```

Comment:
The `identity` is already passed to the `register` function, shouldn't the `keytype` be computed within that function rather than extending its declaration?

Reply:
Ah, this also includes a change to have less constraints on the `identity`. Previously, we had an `IIWebAuthnIdentity` which had information attached about the `KeyType`. Now, we only pass a `SignIdentity` (in preparation of the PIN key _not_ being an `IIWebAuthnIdentity`).

Reply:
Ummm ok. Not sure it makes the code that readable from having a look at this PR but you know better.